homeassistant/components/google_assistant/smart_home.py

```diff
@@ -253,14 +253,15 @@ def async_update(self):
 
 
 @HANDLERS.register('action.devices.SYNC')
-async def async_devices_sync(hass, config, request_id, payload):
+async def async_devices_sync(hass, config, payload):
     """Handle action.devices.SYNC request.
 
     https://developers.google.com/actions/smarthome/create-app#actiondevicessync
     """
-    hass.bus.async_fire(EVENT_SYNC_RECEIVED, {
-        'request_id': request_id
-    })
+    hass.bus.async_fire(
+        EVENT_SYNC_RECEIVED,
+        {'request_id': config.request_id},
```

Comment:
We should not use the config as a place to hold values. We could receive two commands at the same time if different users have it linked.